apache-cassandra-mixin/README.md

```diff
@@ -53,7 +53,7 @@ scrape_configs:
         labels:
           job: integrations/apache-cassandra
           instance: '<your-instance-name>'
-          cluster: '<your-cluster-name>'
+          cassandra_cluster: '<your-cluster-name>'
```

Comment:
Seems like the rename of `cluster` => `cassandra_cluster` is a good move. 

I understand that the `cluster` label is reserved and that is the motivating factor behind this effort, do we have a good understanding of when the label gets populated?

Reply:
@schmikei i believe when you use the integration through the kubernetes plugin then it gets added automatically (i could be wrong).

Reply:
I guess my concern is communicating for people not using the k8s plugin if this will work OOTB. Will look into that a tad bit closer on my 2nd pass :)

Reply:
I guess for people not using the plugin was more of my concern. Which should be more on the integration side I believe.

Reply:
After re-reading this I think it could benefit a little from rephrasing 😅 

My question was more of people who are using cluster currently will be affected by this. It feels like this **IS** a correct approach on renaming to use `cassandra_cluster` and that is what we want users to do. So does that warrant a documentation change somewhere was the primary thing I wanted to get clarification on? Or if we are okay sending the correct implementation with an update to the integration's documentation?

Reply:
Keeping this visible for Grafana reviewers.

Reply:
it would definitely need an docs update - however this would happen automatically as long as we make the changes in the integrations files in cloud-onboarding (agent config), right ? @Dasomeone

Reply:
The change would also need to be made in the scrape configuration, at which point the CI would demand new docs generation as well :)